### PR TITLE
Ucs fixes (bsc#961761)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by vuntz** [comment]
+1

**Comment by aplanas** [comment]
+1

File: crowbar_framework/app/controllers/ucs_controller.rb [hunk 1]
```diff
@@ -303,6 +303,12 @@ def truncate(message)
   def sendXML(xmlString = "")
     uri = URI.parse(@ucs_url)
     http = Net::HTTP.new(uri.host, uri.port)
+    http.use_ssl = False
```
Comment (tboerger) [inline crowbar_framework/app/controllers/ucs_controller.rb]: We are not on python, that should be `false` and not `False`

File: crowbar_framework/app/controllers/ucs_controller.rb [hunk 2]
```diff
@@ -303,6 +303,12 @@ def truncate(message)
...
+    if uri.scheme == "https"
+      http.use_ssl = True
```
Comment (tboerger) [inline crowbar_framework/app/controllers/ucs_controller.rb]: We are not on python, that should be `true` and not `True`

File: crowbar_framework/app/controllers/ucs_controller.rb (hunk 1 above)
Comment (tboerger) [inline crowbar_framework/app/controllers/ucs_controller.rb]: Beside that setting it to false is anyway a waste.

**Pull request merged**